internal/cli/atlas/commonerrors/errors.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 var (
-	errClusterUnsupported = errors.New("cluster update is not supported, try 'atlas cluster upgrade' command")
+	errClusterUnsupported = errors.New("the command supports only M10+ clusters. You can upgrade your cluster by running the 'atlas cluster upgrade' command")
```

Comment:
[q] what's the error the API throws? 1) if it says M10+ only supports, maybe it would be better to forward that error 2) if it does not say that, maybe we should also update the API itself? 

thinking overall if we want to display API specific limitations in the CLI or simply have a specific error API_TIER_ONLY_SUPPORT thrown from the API and have the CLI display it - that way if M10+ turns into M11+ we don't need to update. (not-blocking we can have another task for that in the future so we can improve xp quickly)

Reply:
I like the idea of providing additional user guidance in the error message. We should do this with the Atlas API itself, but that's a project for another day. Left a minor suggestion.